setup.py

```diff
@@ -54,7 +54,7 @@
       install_requires=[
           'setuptools',
           'requests >= 1.0',
-          'ruamel.yaml >= 0.12.4',
+          'ruamel.yaml >= 0.12.4, <= 0.15.2',
```

Comment:
I was using ``0.15.2`` due to some mypy updates which were released in this version, but yeah they didn't help much. I will change to ``ruamel.yaml<=0.15``